### ISSUE TITLE
Edit iframe embed widget view is blank

### ISSUE THREAD

**Issue by kevindherman** [proudcity/wp-proudcity#1069]
When clicking in to edit the iframe embed wiget, the edit view show a blank widget interface.

Here's an example page:
https://www.cityofportorchard.us/wp-admin/post.php?post=2992&action=edit

![screenshot 2017-08-07 10 14 21](https://user-images.githubusercontent.com/6032426/29030568-73ea32b6-7b59-11e7-9c3e-d3ba8e2999fb.png)

**Comment by kevindherman** [comment]
@jlyon @aschmoe this has been reported on SR Parks too.

**Comment by jlyon** [comment]
We are going to switch the iframe widget from WP_Widget_Text (which now has an Editor, as of WP 4.8.0) to WP_Widget_Custom_HTML, which was introduced in WP 4.8.1.

Database migration to be run in Release 1.36.0
```
wp search-replace WP_Widget_Text WP_Widget_Custom_HTML --allow-root
```

**Comment by jlyon** [comment]
@kevindherman do you have the SR parks page? If you send it, I will make sure that it gets cleaned up nicely after the release tomorrow.

Also, can you think of any other pages on client sites that are using the iframe widget.  There's gonna be a little bit of a migration, so I want to make sure to test a few sites after the release.

**Comment by kevindherman** [comment]
here's the sr parks page: https://www.livelifelocally.org/?page_id=3656&preview=true

The PO council video page is a big one. I can't think off the top of my head where else it's relied on. It's not the most popular widget but it's not uncommon either.

**Comment by kevindherman** [comment]
@jlyon since we don't how many people will be effected by this change are there ways to communicate it to the platform in addition to the email? Maybe a message or banner for two business days alerting people if possible.

**Comment by jlyon** [comment]
@kevindherman the frontend of the SR Parks page should currently not show anything, correct?

I don't anticipate there being issues, I just needed another example of an iframe page that is currently displaying something on the frontend but un-editable on the backend.  I found it (https://www.westcarrollton.org/residents/).  I'll test that and make sure it works and then make sure that we're set on these two pages (PO and SR Parks).  I will also make sure it is promintent in the release notes.

If we feel like we want to make this even more prominent, we could temporarily set this as the message on the Did you know? dashboard block.  We should also figure out a way to get the old news feed on the new dashboard.  I created an issue for that.

**Comment by kevindherman** [comment]
@jlyon re: Sr parks, correct.

re: another example, sounds good. If the WC page gets weird I would go with your communication plan but hold off if everything is smooth.

**Comment by jlyon** [comment]
@kevindherman I fixed the three iframe widgets on https://www.cityofportorchard.us/city-council-meeting-videos/ and got rid of the text widgets that were breaking the edit page.

The edit box is now working on the iframe page https://www.livelifelocally.org/?page_id=3656&preview=true

https://www.westcarrollton.org/residents/ came over just fine.

I decided against updating the dashboard  Did you know?  block.
